src/m365/spo/commands/group/group-member-add.ts

```diff
@@ -245,6 +264,30 @@ class SpoGroupMemberAddCommand extends SpoCommand {
     validUserNames.push(JSON.parse(aadUserGetOutput.stdout).userPrincipalName);
   }
 
+  private async aadGroupGet(options: Options, userIdentifier: string, logger: Logger, validUserNames: string[]): Promise<void> {
+    if (this.verbose) {
+      logger.logToStderr(`Get UPN from Azure AD for group ${userIdentifier}`);
+    }
+
+    const aadUserGetCommandoptions: AadGroupGetCommandOptions = {
+      ...(options.aadGroupId && { id: userIdentifier }),
+      ...(options.aadGroupName && { title: userIdentifier }),
+      output: 'json',
+      debug: options.debug,
+      verbose: options.verbose
+    };
+
+    const aadGroupGetOutput: CommandOutput = await Cli.executeCommandWithOutput(AadGroupGetCommand as Command, { options: { ...aadUserGetCommandoptions, _: [] } });
+
+    if (this.debug) {
+      logger.logToStderr(aadGroupGetOutput.stderr);
+    }
+
+    logger.log(JSON.parse(aadGroupGetOutput.stdout));
```

Comment:
I don't think we need this log. Now we will get two outputs when adding aad group as member
```
{
  "id": "0d718612-8407-4d6b-833c-6891a553354f",
  "deletedDateTime": null,
  "classification": null,
  "createdDateTime": "2021-04-11T21:34:20Z",
  "creationOptions": [
    "ProvisionGroupHomepage",
    "HubSiteId:00000000-0000-0000-0000-000000000000",
    "SPSiteLanguage:1033"
  ],
  "description": "xxxx",
  "displayName": "aaa1",
  "expirationDateTime": null,
  "groupTypes": [
    "Unified"
  ],
  "isAssignableToRole": null,
  "mail": "testSetDescription@tenanttocheck.onmicrosoft.com",
  "mailEnabled": true,
  "mailNickname": "testSetDescription",
  "membershipRule": null,
  "membershipRuleProcessingState": null,
  "onPremisesDomainName": null,
  "onPremisesLastSyncDateTime": null,
  "onPremisesNetBiosName": null,
  "onPremisesSamAccountName": null,
  "onPremisesSecurityIdentifier": null,
  "onPremisesSyncEnabled": null,
  "preferredDataLocation": null,
  "preferredLanguage": null,
  "proxyAddresses": [
    "SPO:SPO_0445d096-e0ae-4c00-b37e-0dee0e9a824a@SPO_2942bb31-1d49-4da6-8d3d-d0f9e1141486",
    "SMTP:testSetDescription@tenanttocheck.onmicrosoft.com"
  ],
  "renewedDateTime": "2022-07-17T14:40:27Z",
  "resourceBehaviorOptions": [],
  "resourceProvisioningOptions": [],
  "securityEnabled": false,
  "securityIdentifier": "S-1-12-1-225543698-1298891783-2439527555-1328894885",
  "theme": null,
  "visibility": "Private",
  "onPremisesProvisioningErrors": []
}
[
  {
    "AllowedRoles": [
      0
    ],
    "CurrentRole": 0,
    "DisplayName": "aaa1 Members",
    "Email": "testSetDescription@tenanttocheck.onmicrosoft.com",
    "InvitationLink": null,
    "IsUserKnown": true,
    "Message": null,
    "Status": true,
    "User": "c:0o.c|federateddirectoryclaimprovider|0d718612-8407-4d6b-833c-6891a553354f"
  }
]
```

This will produce strange behavior when `--output text` would be used. (we will get space for the first log)
![image](https://user-images.githubusercontent.com/58668583/211686879-57f8b043-d99e-439b-9389-aff738088be2.png)